sisl/viz/plotly/plot.py

```diff
@@ -694,8 +703,10 @@ def __init__(self, *args, H = None, attrs_for_plot={}, only_init=False, presets=
         except Exception as e:
             if self._debug:
                 raise e
-            else:
-                print("The plot has been initialized correctly, but the current settings were not enough to generate the figure.\n (Error: {})".format(e))
+            # TODO this gets printed alot, could you please fix this.
+            # I have no idea what this means, but it seems like an error should be thrown regardless.
+            # Many tests execute this line while still succeeding, I don't get that?
+            print(f"The plot has been initialized correctly, but the current settings were not enough to generate the figure.\nError: {e}")
```

Comment:
Ok, but then you should issue warnings instead of printing. Generally one should never print anything out.

warnings are good because they can be filtered out.
I have tried to clean up in sisl, and not that many print's are there now.

Reply:
Ok :+1: Can you change it? I can't act on this PR.

Reply:
I'll replace all print to warn/info were I think appropriate, then you may have a look at these changes to see if they are ok.

Reply:
Done, could you have a look? I sort of guessed what should be information, and what should be warnings, did I get it right?